### PR TITLE
Build

### PR DESCRIPTION
**Pull request by buzzdecafe**
build the latest tag in one easy `npm run docs`

### CODE REVIEW

**Comment by davidchambers** [comment]
Could you squash and `git rebase master`?

**Comment by buzzdecafe** [comment]
squashed

File: .gitignore [hunk 1]
```diff
@@ -1 +1,3 @@
 tmp
+bower_components
+out
```
Comment (davidchambers) [inline .gitignore]: I would write this:

    /bower_components/
    /out/
    /tmp/

Changes:

  - leading slash to anchor to project root;
  - trailing slash for each directory; and
  - alphabetical order.

File: bower.json [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    "bower_components",
+    "test",
+    "tests",
+    "examples",
+    "logo"
+  ],
+  "devDependencies": {
+    "bootstrap": "~3.3.1"
+  }
+}
```
Comment (davidchambers) [inline bower.json]: Why is this file necessary?

Comment (buzzdecafe) [inline bower.json]: docs rely on bootstrap & jquery

File: package.json [hunk 3]
```diff
@@ -0,0 +1,51 @@
+{
+  "name": "ramda.github.io",
+  "version": "0.0.0",
```
Comment (davidchambers) [inline package.json]: This differs from the version in __bower.json__.

File: scripts/_helpers.js [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+var rx = /\D/g;
+
+var opts = {
+  url: 'https://api.github.com/repos/ramda/ramda/tags',
+  headers: {'user-agent': 'node.js'}
+};
+var outDir = './tmp';
+
+function byName(a, b) { 
+  return parseInt(b.name.replace(rx, ''), 10) - parseInt(a.name.replace(rx, ''), 10);
```
Comment (davidchambers) [inline scripts/_helpers.js]:
```javascript
> parseInt('0.10.10'.replace(/\D/g, ''), 10)
1010
> parseInt('9.9.9'.replace(/\D/g, ''), 10)
999
```

File: bower.json (hunk 2 above)
Comment (davidchambers) [inline bower.json]: Is there a CDN-hosted version of Bootstrap we could use?

Comment (buzzdecafe) [inline bower.json]: i think we need it for the build:

ramda.less: `@import "bower_components/bootstrap/less/bootstrap"`
variables.less: `@icon-font-path: "fonts/";`

I suggest we table this. We can come back and improve on it later.

File: package.json (hunk 3 above)
Comment (buzzdecafe) [inline package.json]: nice catch. just copied bower.json from ramda

File: scripts/_helpers.js (hunk 4 above)
Comment (buzzdecafe) [inline scripts/_helpers.js]: curses!

File: .gitignore (hunk 1 above)
Comment (buzzdecafe) [inline .gitignore]: :+1:

File: bower.json (hunk 2 above)
Comment (buzzdecafe) [inline bower.json]: may be able to use npm and dispense with bower altogether. will investigate tomorrow

**Comment by buzzdecafe** [comment]
was able to eliminate bower. i think this is good to go. i :cow2: myself

**Comment by davidchambers** [comment]
> was able to eliminate bower.

Sweet!

**Comment by buzzdecafe** [comment]
once this is in, I think we are good to remove grunt from ramda/ramda :smile_cat:

File: scripts/_helpers.js [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+function fetchOpts(tags) {
+  var tag = head(JSON.parse(tags).sort(byName));
+  return assoc('url', tag.tarball_url, opts)
+}
+
+// [String] -> [String]
+var zeroPad = R.map(function pad(n) { return n.length === 3 ? n : pad('0' + n); });
+
+// String -> Number
+versionToNum = R.pipe(
```
Comment (davidchambers) [inline scripts/_helpers.js]: Missing `var`?

File: scripts/_helpers.js [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+  return versionToNum(b.name) - versionToNum(a.name);
+}
+
+function fetchOpts(tags) {
+  var tag = head(JSON.parse(tags).sort(byName));
+  return assoc('url', tag.tarball_url, opts)
+}
+
+// [String] -> [String]
+var zeroPad = R.map(function pad(n) { return n.length === 3 ? n : pad('0' + n); });
```
Comment (davidchambers) [inline scripts/_helpers.js]: `zeroPad` should have type `String -> String`. You could do the mapping in `versionToNum`.

**Comment by davidchambers** [comment]
I made a couple of comments, but LGTM otherwise. Let's merge this and remove Grunt from ramda/ramda!

**Comment by buzzdecafe** [comment]
thank you for the review! I appreciate your almost fanatical attention to software quality.

I'm gonna merge this, then I want the pleasure of making a PR to remove Grunt on ramda/ramda.
